fontc/src/main.rs

```diff
@@ -478,6 +484,30 @@ fn add_cmap_be_job(
     Ok(())
 }
 
+fn add_post_be_job(
+    change_detector: &mut ChangeDetector,
+    workload: &mut Workload,
+) -> Result<(), Error> {
+    if change_detector.post_be_change() {
+        let mut dependencies = HashSet::new();
+        dependencies.insert(FeWorkIdentifier::FinalizeStaticMetadata.into());
```

Comment:
I think your minimalist post v3 does not have any dependencies, does it? It's a constant string, basically

Reply:
That's right, will remove.